integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/putmessage/StrategyFactory.java

```diff
@@ -38,10 +46,15 @@ public MessageStrategyFactory getFactory(ServiceRecord serviceRecord) {
             return eduMessageStrategyFactory;
         } else if ("POST_VIRKSOMHET".equalsIgnoreCase(serviceRecord.getServiceIdentifier())) {
             return postVirksomhetStrategyFactory;
+        } else if ("DPI".equalsIgnoreCase(serviceRecord.getServiceIdentifier())){
```

Comment:
Jeg tenker det kommer som endel av plugin arkitekturen. For øyeblikket er det sikkert greit de er konstanter i koden siden det mest sannsynligvis vil bli gjort om på da.